components/ws-daemon/pkg/daemon/daemon.go

```diff
@@ -58,12 +58,17 @@ func NewDaemon(config Config, reg prometheus.Registerer) (*Daemon, error) {
 		return nil, err
 	}
 
+	cgroupV2IOLimiter, err := cgroup.NewIOLimiterV2(config.IOLimit.WriteBWPerSecond.Value(), config.IOLimit.ReadBWPerSecond.Value(), config.IOLimit.WriteIOPS, config.IOLimit.ReadIOPS)
+	if err != nil {
+		return nil, err
+	}
+
 	cgroupPlugins, err := cgroup.NewPluginHost(config.CPULimit.CGroupBasePath,
 		&cgroup.CacheReclaim{},
-		&cgroup.FuseDeviceEnablerV1{},
-		&cgroup.FuseDeviceEnablerV2{},
 		cgroupV1IOLimiter,
-		cgroup.NewIOLimiterV2(config.IOLimit.WriteBWPerSecond.Value(), config.IOLimit.ReadBWPerSecond.Value(), config.IOLimit.WriteIOPS, config.IOLimit.ReadIOPS),
+		cgroupV2IOLimiter,
+		&cgroup.FuseDeviceEnablerV1{},
+		//&cgroup.FuseDeviceEnablerV2{},
```

Comment:
How about using this function to determine which cgroup should use?
https://github.com/gitpod-io/gitpod/blob/e5c37ddbe1e8e994008f0ff12893d5e8c740848c/test/pkg/integration/common/cgroup.go#L13

Reply:
Fixed